utils/cas_version_gen

```diff
@@ -29,11 +29,13 @@ error() {
 if [[ -d "$SOURCES_DIR/.git" ]] && which git &>/dev/null &&\
         (cd "$SOURCES_DIR" && git rev-parse --is-inside-work-tree &>/dev/null); then
     if [[ ! -r "$MANUAL_VERSION_INPUT" ]]; then
-        error "can't read version input file"
+        error "can't read version input file '$MANUAL_VERSION_INPUT'"
     fi
     . "$MANUAL_VERSION_INPUT"
     if [[ ! "$CAS_VERSION_MAIN" || ! "$CAS_VERSION_MAJOR" || ! "$CAS_VERSION_MINOR" ]]; then
-        error "wrong version input file format"
+        error "'$MANUAL_VERSION_INPUT' - wrong version input file format;"\
+            "file should contain CAS_VERSION_MAIN, CAS_VERSION_MAJOR and CAS_VERSION_MINOR"\
```

Comment:
You forgot spaces before breaking the lines.

Reply:
No, each quoted block is separated by a space. "\" escapes only a new line.